includes/dublin_core.inc

```diff
@@ -152,7 +152,7 @@ class DublinCore {
    * @return DublinCore
    *   The instantiated object.
    */
-  public static function importFromXMLString($dc_xml) {
+  public static function importFromXmlString($dc_xml) {
```

Comment:
This change affects (nominally because php is case insensitive for functions/methods http://fr.php.net/manual/en/functions.user-defined.php). I tracked this use to almost every SP that we have. (webarchive, large image, pdf, you name it)